conans/client/installer.py

```diff
@@ -231,7 +231,7 @@ def raise_package_not_found_error(conan_file, conan_ref, package_id, out, record
 ''' % (conan_ref, settings_text, options_text, package_id)
     out.warn(msg)
     recorder.package_install_error(PackageReference(conan_ref, package_id),
-                                   INSTALL_ERROR_MISSING, msg, remote=remote_url)
+                                   INSTALL_ERROR_MISSING, msg, remote_name=remote_url)
```

Comment:
This ``remote_name=remote_url`` reads very weird.

Reply:
Actually it is always called with None, I'll remove it